### PR TITLE
Validate multiple emails  

### PR DESCRIPTION
**Pull request by monishdeb**
Validate multiple emails while testing on compose screen: 

nyss-mosaico

### CODE REVIEW

**Comment by monishdeb** [comment]
@colemanw @lcdservices can you please review this patch !!

**Comment by mattwire** [comment]
@monishdeb Needs rebase

File: ang/crmMosaico.bootstrap/BlockPreview.html [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
-    type="email"
+    type="text"
     class="form-control margin-bottom-10"
     ng-model="testContact.email"
     placeholder="example@example.org" />
   <!--fa-paper-plane-->
-  <button type="button" class="btn btn-sm btn-primary" title="{{ crmMailing.$invalid || !testContact.email ? ts('Complete all required-mark fields first') : ts('Send test message to %1', {1: testContact.email}) }}" ng-disabled="crmMailing.$invalid || !testContact.email" ng-click="doSend({email: testContact.email})">
-    {{:: ts('Send test') }}
+  <button class="btn btn-sm btn-primary" title="{{crmMailing.$invalid || !testContact.email ? ts('Complete all required-mark fields first') : !validateMultipleEmail(testContact.email) ? ts('Not a valid email address') : ts('Send test message to %1', {1: testContact.email})}}" ng-disabled="crmMailing.$invalid || !testContact.email || !validateMultipleEmail(testContact.email)" ng-click="doSend({email: testContact.email})">
+    {{ts('Send test')}}
```
Comment (seamuslee001) [inline ang/crmMosaico.bootstrap/BlockPreview.html]: I think the change on this line should be reverted right @colemanw ?

**Comment by monishdeb** [comment]
@mattwire I have done the rebase .. please have a look now.

**Comment by mattwire** [comment]
Thanks @monishdeb I'm merging on the basis that we (someone) will do quite a bit of testing before the next release.

**Pull request merged**